pkg/injector/pod_patch.go

```diff
@@ -124,12 +126,14 @@ func (i *injector) getPodPatchOperations(ar *v1beta1.AdmissionReview,
 	}
 
 	patchOps := []PatchOperation{}
+	envPatchOps := []PatchOperation{}
 	var path string
 	var value interface{}
 	if len(pod.Spec.Containers) == 0 {
 		path = "/spec/containers"
 		value = []corev1.Container{*sidecarContainer}
 	} else {
+		envPatchOps = setDaprEnvVars(pod.Spec.Containers, "/spec/containers")
```

Comment:
I would define const for `/spec/containers` which is used in multiple places. then I will not pass this argument.